cmdstanpy/utils/stancsv.py

```diff
@@ -60,13 +60,13 @@ def csv_bytes_list_to_numpy(
     """Efficiently converts a list of bytes representing whose concatenation
     represents a CSV file into a numpy array. Includes header specifies
     whether the bytes contains an initial header line."""
+    if not csv_bytes_list:
+        return np.empty((0,))
```

Comment:
Not sure it's worth complicating the logic, but it would make sense to me if a completely empty list was illegal when `includes_header=True`. I don't think that would ever happen on a valid csv file, though